addons/web_editor/static/lib/odoo-editor/test/utils.js

```diff
@@ -600,4 +600,11 @@ export function triggerEvent(
     return ev;
 }
 
+export async function fetchFileContents(url) {
```

Comment:
Well I'd have named this `load` or `read` but nico would hate me for that. Though I still feel like `fetchFileContents` is maybe a tad too specific. What do you think of `readFromFile` ? It would look like this:
```js
await testEditor(BasicEditor, {
    contentBefore: '<p>[]</p>',
    stepFunction: async editor => {
        const clipboard = await readFromFile('/web_editor/static/lib/odoo-editor/test/spec/html_tables/excel_clipboard.html');
        await pasteHtml(editor, clipboard);
    },
    contentAfter: await readFromFile('/web_editor/static/lib/odoo-editor/test/spec/html_tables/excel_odoo.html'),
}
```

Reply:
LGTM